pybossa/auth/task.py

```diff
@@ -41,7 +41,7 @@ def _read(self, user, task=None):
         return user.is_authenticated
 
     def _update(self, user, task):
-        return self._only_admin_or_subadminowners(user, task)
+        return self._only_admin_or_owners(user, task)
```

Comment:
changing the _update implementation will affect all code calling ```ensure_authorized_to('update', project)```. Probably this is not what we want.

Reply:
This is under task auth, hence would be applied to task and won't affect all code.  call to `ensure_authorized_to('update', project)` would route to project auth [here](https://github.com/bloomberg/pybossa/blob/main/pybossa/auth/project.py#L61)